db/sql/sql.go

```diff
@@ -528,17 +538,28 @@ func makeStateColumns(s *schema.Schema) (cols []string) {
 	return cols
 }
 
-func makeJoin(s *schema.Schema, tableName string, q sq.SelectBuilder) sq.SelectBuilder {
+func makeJoin(s *schema.Schema, tableName string, q sq.SelectBuilder, aliasRelationPropertyPrefix string, aliasTableName string) sq.SelectBuilder {
```

Comment:
I'm not sure we need aliasTableName..
How about just pass newAliasTableName as tableName and omit l543-545